src/components/spreadsheet/spreadsheet.ts

```diff
@@ -378,6 +380,15 @@ export class Spreadsheet extends Component<SpreadsheetProps, SpreadsheetChildEnv
       raiseError: (text, cb) => this.notificationStore.raiseError(text, cb),
     } satisfies Partial<SpreadsheetChildEnv>);
 
+    if (!("isSmall" in this.env)) {
```

Comment:
I guess the idea is to rely on `env.isSmall` defined by odoo ?

Reply:
yup 👍